packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx

```diff
@@ -152,21 +180,43 @@ export const DeckLayout = ({
         {/* Dialog overlay to dismiss dialogs. */}
         <Main.Overlay />
 
-        {/* Main content surface. */}
-        {layoutMode === 'deck' && layoutParts.main && layoutParts.main.length > 0 && (
+        {/* No content. */}
+        {isEmpty && (
+          <Main.Content>
+            <ContentEmpty />
+          </Main.Content>
+        )}
+
+        {/* TODO(burdon): Preserve DOM structure between solo/deck mode. */}
+        {!isEmpty && (
           <Main.Content bounce classNames={['grid', 'block-end-[--statusbar-size]']}>
-            <div role='none' className='relative'>
```

Comment:
I tried removing this node in my PR and it complicated things — does Deck still work (scrolls horizontally, responsive layout is correct) when this is removed?

Reply:
it seems to work -- my goal was to make the deck and solo keep as much similarity in dom as possible.